apartmenttherapy/order-service
# Elaine/bug data corruption

#### What does this PR do & why?
This fixes a bug where the `deliveryAddress` was making it into the `Order` table in the db. I also refactored a bunch of stuff to behave better.

#### How can this be manually tested?
- Tests should run.
- Just run through an order lifecycle, making sure nothing other than the following fields make it into the `Order` table:
```
export const ORDER_DESC = [
  'changeLog',
  'chargeId',
  'closedReason',
  'createdAt',
  'customerUserId',
  'deliveryAddressId',
  'deliveryOption',
  'id',
  'offerAmount',
  'orderStatus',
  'productId',
  'productType',
  'sellerUserId',
  'shippingAmount',
  'totalAmount',
  'updatedAt'
];
```

#### Any questions or concerns about this change?
*(Any significant changes you're introducing? What side effects might this have? What other parts of the app could this affect?)*


#### Deployment Notes
*(e.g. Does this require a migration or special infrastructure change? Clearing the cache? Third party service?)*


#### Link to story card and other documentation
https://app.asana.com/0/694906336753887/1112905843616182

#### GIF


#### Have you remembered to...?
